src/components/Tabs/TabsNav.test.tsx

```diff
@@ -27,30 +28,101 @@ afterEach(() => {
-});
+      expect(list.length).toBe(1);
+    });
+
+    test('should apply Tab class only to the Link elements', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const link = container.getElementsByClassName(styles.Tab);
+      expect(link.length).toBe(3);
+    });
+
+    test('should apply Selected class on default selected link', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const selectedLink = container.getElementsByClassName(styles.Selected);
+
+      expect(selectedLink[0]).toHaveTextContent(MOCK_TABS[0].text);
+    });
+
+    test('should apply Selected class only to the selected link', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const selectedLink = container.getElementsByClassName(styles.Selected);
+
+      expect(selectedLink.length).toBe(1);
+      expect(selectedLink[0]).not.toHaveTextContent(MOCK_TABS[1].text);
+      expect(selectedLink[0]).not.toHaveTextContent(MOCK_TABS[2].text);
+    });
+
+    test('should not apply Selected class on any Link by default', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} location={undefined} />);
+      const selected = container.getElementsByClassName(styles.Selected);
+
+      expect(selected.length).toBe(0);
+    });
+  });
+
+  describe('list', () => {
+    test('should render correct number of links', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const tab = container.getElementsByClassName(styles.Tab);
+
+      expect(tab.length).toEqual(MOCK_TABS.length);
+    });
+
+    test('should render links in the correct order', () => {
+      const MOCK_TABS: TabNav[] = [
+        { text: 'Tab 1,', to: '/1' },
+        { text: 'Tab 2,', to: '/2' },
+        { text: 'Tab 3', to: '/3' }
+      ];
+
+      const { container } = render(<TabsNav tabs={MOCK_TABS} location={MOCK_TABS[0].to} />);
+
+      expect(container.textContent).toBe('Tab 1,Tab 2,Tab 3');
+    });
+  });
+
+  describe('link/tab', () => {
+    test('should apply replace prop to Link', () => {
+      render(<TabsNav {...DEFAULT_PROPS} tabs={[MOCK_TABS[0]]} />);
+
+      expect(mockLink).toHaveBeenCalledTimes(1);
+      expect(mockLink).toHaveBeenCalledWith(expect.objectContaining({ replace: true }));
+    });
+
+    test('should not apply replace prop to Link by default', () => {
+      render(<TabsNav {...DEFAULT_PROPS} tabs={[MOCK_TABS[1]]} />);
+
+      expect(mockLink).toHaveBeenCalledTimes(1);
+      expect(mockLink).toHaveBeenCalledWith(expect.objectContaining({ replace: undefined }));
+    });
+
+    test('should apply and display tab label text', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const tab = container.getElementsByClassName(styles.Tab);
+
+      expect(tab[0]).toHaveTextContent(MOCK_TABS[0].text);
+      expect(screen.getByText(MOCK_TABS[0].text)).toBeInTheDocument();
+    });
```

Comment:
This is covered by `should render links in the correct order`

Reply:
Good shout, updated !